Formula/kubernetes-service-catalog-client.rb

```diff
@@ -0,0 +1,28 @@
+class KubernetesServiceCatalogClient < Formula
+  desc "Consume Services in k8s using the OSB API"
+  homepage "https://svc-cat.io/"
+  url "https://github.com/kubernetes-incubator/service-catalog.git",
+      :tag => "v0.1.23",
+      :revision => "87a5db0e1e0359ce372037a235ce4448944c8611"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["NO_DOCKER"] = "1"
+    dir = buildpath/"src/github.com/kubernetes-incubator/service-catalog"
+
+    dir.install buildpath.children
+
+    cd dir do
+      ldflags = "-s -w -X github.com/kubernetes-incubator/service-catalog/pkg.VERSION=v#{version}"
+      system "go", "build", "-ldflags", ldflags, "-o", "bin/svcat", "./cmd/svcat"
+      bin.install "bin/svcat"
+    end
+  end
+
+  test do
+    version_output = shell_output("#{bin}/svcat version --client 2>&1")
+    assert_match "Client Version: v0.1.23", version_output
```

Comment:
Don't hardcode the version here. Use `#{version}`, but ideally something that tests a fuller level of functionality would be nice.

Reply:
Thanks @DomT4 . Fixed it.